checkers-app/src/services/api.ts

```diff
@@ -0,0 +1,14 @@
+import axios from "axios"
+import { FactChecker } from "../types/factChecker"
+
+// const BASE_URL = import.meta.env.DEV ? "http://127.0.0.1:5000" : "PROD ENDPOINT"
```

Comment:
No issues here but in case this wasn't clear previously, I'm using the rewrites functionality in firebase.json to do routing as per https://firebase.google.com/docs/hosting/functions#direct-requests-to-function

firebase.json
```
    "rewrites": [
      {
        "source": "/api/**",
        "function": {
          "functionId": "apiHandler",
          "region": "asia-southeast1"
        }
      },
      ...
    ]
```

Reply:
Okay thanks! I need to read up more on the JSON configurations :)